tv/interfaces/speaker/index.ts

```diff
@@ -0,0 +1,46 @@
+export interface PublicLink {
+  id: string;
+  relativeUrl: string;
+  versionHash: string;
+}
+
+export interface AssetToPublicLink {
+  results: PublicLink[];
+}
+
+export interface AssetResult {
+  id: string;
+  fileName: string;
+  assetToPublicLink: AssetToPublicLink;
+}
+
+export interface Image {
+  results: AssetResult[];
+}
+
+export interface Result {
```

Comment:
`Result` seems a bit generic for a name. Other interfaces files have something like `SpeakerResult` for that interface. Could it be renamed?

Reply:
Fixed